README.md

```diff
@@ -4,9 +4,9 @@ A Concourse CI resource to build new [Amazon Machine Images (AMI) via Packer](ht
 
 ## Source Configuration
 
-- `aws_access_key_id`: *Required.* Your AWS access key ID.
+- `aws_access_key_id`: *Optional.* Your AWS access key ID.
 
-- `aws_secret_access_key`: *Required.* Your AWS secret access key.
+- `aws_secret_access_key`: *Optional.* Your AWS secret access key.
```

Comment:
Thus far, the documentation has only drawn attention to required fields. Perhaps drop the bold "Optional" and add a note at the end about using the standard aws-cli fallbacks (including the instance profile)?